src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/ReadsPipelineSpark.java

```diff
@@ -70,7 +70,7 @@
     @Argument(doc = "whether to perform alignment using BWA-MEM", shortName = "align", fullName = "align", optional = true)
     private boolean align;
 
-    @Argument(doc = "the known variants", shortName = "knownSites", fullName = "knownSites", optional = false)
+    @Argument(doc = "the known variants", shortName = "known", fullName = "known-sites", optional = false)
```

Comment:
Update program group and add an example command please.

Reply:
Fixed